roles/common/tasks/main.yml

```diff
@@ -15,3 +15,9 @@
     regexp: '^127\.0\.0\.1'
     line: '127.0.0.1 {{ ansible_hostname }} localhost'
   tags: ['not-on-docker']
+
+- name: Set custom bash prompt for per-environment nodes
+  template:
+    dest: /etc/profile.d/bash_prompt.sh
+    src: bash_prompt.sh
```

Comment:
We prefer our template files to live in the relative path within `templates/` that matches the destination path on the host. Can you move this to `templates/etc/profile.d/bash_prompt.sh`?